xlators/cluster/ec/src/ec-dir-write.c

```diff
@@ -228,11 +228,14 @@ ec_manager_create(ec_fop_data_t *fop, int32_t state)
         case -EC_STATE_DISPATCH:
         case -EC_STATE_PREPARE_ANSWER:
         case -EC_STATE_REPORT:
+            cbk = fop->answer;
+
+            GF_ASSERT(cbk != NULL);
```

Comment:
That's not guaranteed. `cbk` may be NULL is there are not enough consistent bricks or the operation has failed early for some other reason.